resources/views/admin/dashboards.blade.php

```diff
@@ -0,0 +1,91 @@
+@extends('layouts.app')
+
+@section('content')
+
+{{-- temporaly use for front end coding--}}
+               <!-- Bootstrap -->
+               <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.3.0/dist/css/bootstrap.min.css" rel="stylesheet" integrity="sha384-9ndCyUaIbzAi2FUVXJi0CjmCapSmO7SnpJef0486qhLnuZ2cdeRhO02iuK6FUUVM" crossorigin="anonymous">
```

Comment:
if you can see my comment in your other PR about having styles like this in a separate section

https://github.com/kredo-school/14th_explore_ease/pull/14#discussion_r1383161110